docs/events.md

```diff
@@ -0,0 +1,88 @@
+# Player Events
+A player event is a string within the player that represents an event.
+
+## Overview
+
+Player events consist of two event types:
+
+ 1. HTML5 events - various events sent by the browser when handling media embedded using the `<video>` element.
+ 2. Kaltura player custom events - special events that indicate changes in the state of the player, which do not exist in the html5 video event list and are related to the integral behavior of the player, such as ads, fullscreen and subtitles events.
+
+Only DOM elements may be true EventTargets. Inside the player, we use fake events targets (fake-event-target.js), and fake events (fake-event) in order to dispatch events not from a DOM element.
+
+## Importing the Events List
+
+ ```javascript
+ import {HTML5_EVENTS as Html5Events, CUSTOM_EVENTS as CustomEvents} from 'path/to/../event/events'
```

Comment:
I don't see this is useful as we export this in the API as `player.Event`, which is more important to show case here